src/world/area_tik/tik_03/86F0F0.c

```diff
@@ -2,8 +2,49 @@
 
 #include "world/common/UnkPosFunc2.inc.c"
 
-INCLUDE_ASM(s32, "world/area_tik/tik_03/86F0F0", func_80240960_86F460);
+void partner_set_goal_pos(s32, s32);
+extern s32 D_80242EBC_8719BC;
+extern Npc* wPartnerNpc;
+extern s32 D_80242930_871430[4];
 
-INCLUDE_ASM(s32, "world/area_tik/tik_03/86F0F0", func_802409C0_86F4C0);
+ApiStatus func_80240960_86F460(Evt* script, s32 isInitialCall) {
+    PlayerStatus* player = &gPlayerStatus;
+    u32 i;
```

Comment:
We like to have an empty line between the declarations at the start of a the function and the rest of the function body